### ISSUE TITLE
SwiftMail authentication fails

### ISSUE THREAD

**Issue by Shayne38** [swiftmailer/swiftmailer#1046]
| Q                   | A
| ------------------- | -----
| Bug report?     No    | yes/no
| Feature request? No   | yes/no
| RFC?                | yes/no
| How used?           | Standalone/Symfony/3party
| Swiftmailer version | 3.3.3
| PHP version         | 5

### Observed behaviour
There was a problem communicating with SMTP: Authentication failed using username

**Comment by tomzx** [comment]
Hello,

This is not an issue with Swiftmailer. Apparently your SMTP configuration is wrong and you username/password configuration is incorrect.

**Comment by Shayne38** [comment]
That is what I thought at first. For smtp server I tried the name and also the IP Address which I know are correct. For user name and password I entered the correct credentials several times with no luck. I also made that user a domain admin to rule out a permissions problem and that didn’t help. I then tried using my user name and password and that also failed when I did check SwiftMail configuration. Where can I get documentation on SwiftMail configuration.

Shayne Zwickey
Systems\Network Administrator
Direct: 612-334-9137
PRECISION ASSOCIATES, INC.
